packages/sync-engine/scripts/sigma-sync-local.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Local Sigma sync test script.
```

Comment:
this can probably be removed now that we have `test-integration-sigma.sh` or does it still have value?